### PR TITLE
Project 1: create animals table

### PR DESCRIPTION
**Pull request by cmolinan**
-- Find all animals whose name ends in "mon".
SELECT * FROM animals WHERE name like '%mon';
image

-- List the name of all animals born between 2016 and 2019.
SELECT * FROM animals WHERE date_of_birth between '2016-01-01' AND '2019-12-31';
image

-- List the name of all animals that are neutered and have less than 3 escape attempts.
SELECT * FROM animals WHERE neutered AND escape_attempts < 3 ;
image

-- List the date of birth of all animals named either "Agumon" or "Pikachu".
SELECT name, date_of_birth  FROM animals WHERE name='Agumon' or name='Pikachu'
image

-- List name and escape attempts of animals that weigh more than 10.5kg
SELECT name, escape_attempts FROM animals WHERE weight_kg >10.5
image

-- Find all animals that are neutered.
SELECT * FROM animals WHERE neutered
image

-- Find all animals not named Gabumon.
SELECT * FROM animals WHERE name not like 'Gabumon'
image

-- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
SELECT * FROM animals WHERE weight_kg between 10.4 and 17.3 
image

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Carlos, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect gitflow. Kindly note that the gitflow of this project should have merge request **from feature into development**. Kindly use the correct gitflow and request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Bondok6 — COMMENTED** [review]
## Hi @cmolinan 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has the wrong Gitflow. Your Pull Request should be from the (feature-branch) `create-animals-table` into the (development) `dev` branch.

[Changing the base branch of a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by cmolinan** [comment]
@rloterh @Bondok6 , Oh, sorry for the mistake. I forgot to change the default branch in this repository. It is already corrected !

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+  id     INT GENERATED ALWAYS AS IDENTITY,
+  name    varchar,
+  date_of_birth  date,
+  escape_attempts int,
+  neutered  bit,
+  weight_kg real
 );
```
Comment (Bondok6) [inline schema.sql:10]: [OPTIONAL] I suggest to mark the id column as primary key `PRIMARY KEY (id)` 💯

File: README.md [hunk 2]
```diff
@@ -1,48 +1,30 @@
...
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
+## Acknowledgments 
+This webApp responds to a request from Microverse, which is based on the original idea of Nelson Sakwa, according to the Creative Commons License
+## �� License
 This project is [MIT](./MIT.md) licensed.
```
Comment (Bondok6) [inline README.md:30]: - [OPTIONAL] Your README file looks great 👍 However, The MIT Link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @cmolinan 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Correct GitHub Flow ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (cmolinan) [reply schema.sql:10]: @Bondok6 I forgot it.   I already added it. 
Thanks for let me know  !
image

File: README.md (hunk 2 above)
Comment (cmolinan) [reply README.md:30]: @Bondok6 I forgot, but I already added it.  
Thanks !

**Pull request merged**